core/css.js

```diff
@@ -17,1007 +17,904 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import * as Blockly from 'blockly/core';
+
+const styles = `
+  .blocklySvg {
+    background-color: var(--colour-workspace);
+    outline: none;
+    overflow: hidden;  /* IE overflows by default. */
+    position: absolute;
+    display: block;
+  }
 
-/**
- * @fileoverview Inject Blockly's CSS synchronously.
- * @author fraser@google.com (Neil Fraser)
- */
-'use strict';
-
-/**
- * @name Blockly.Css
- * @namespace
- */
-goog.provide('Blockly.Css');
-
-goog.require('Blockly.Colours');
+  /* Necessary to position the drag surface */
```

Comment:
I filed #34 to track this and other cleanup; I suspect a fair bit of this can be deleted, but figured it was better to bring it all over for now and clean up after.